CONTRIBUTING.md

```diff
@@ -46,6 +48,15 @@ This guide will help you get started and outline some things you should know whe
 * See [here](./documentation/development/testing.md) for how to run the tests
 
 
+## Developing
+
+* update depependencies: `glide up`
+* add a new Go library:
+  * update dependencies in a separate PR
+  * `glide get <package name>`
+  * add the source code of the added library, which is located in [vendor](vendor), to Git
```

Comment:
Thats not clear to me. Thoughts on mentioning that dependencies are checked into source control in the vendor folder as the first bullet above update dependencies?

Then this bullet would simple be commit the updated glide.yaml / glide.lock / vendor folder.